### PR TITLE
TRUNK-5033 Public static fields should be final

### PR DESCRIPTION
**Pull request by dilekamadushan**
JIRA TICKET LINK- https://issues.openmrs.org/browse/TRUNK-5033?filter=-1

TRUNK-5033 Public static fields should be final

I modified public static variables into public static final

Checklist: I completed these to help reviewers :)

My pull request only contains ONE single commit
(the number above, next to the 'Commits' tab is 1).

My IDE is configured to follow the code style of this project.

I have added tests to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

I ran mvn clean package right before creating this pull request and
added all formatting changes to my commit.

All new and existing tests passed.

My pull request is based on the latest changes of the master branch.

### CODE REVIEW

File: api/src/main/java/org/openmrs/util/OpenmrsConstants.java [hunk 1]
```diff
@@ -173,7 +170,7 @@ public static String getOpenmrsProperty(String property) {
 	 * The name of the runtime property that a user can set that will specify whether the database
```
Comment (teleivo) [inline api/src/main/java/org/openmrs/util/OpenmrsConstants.java]: the APPLICATION_DATA_DIRECTORY_RUNTIME_PROPERTY can also be `static final`

Comment (dilekamadushan) [reply api/src/main/java/org/openmrs/util/OpenmrsConstants.java]: @teleivo I made the necessary changes. Feel free to give feedback. Thanks

**Comment by dilekamadushan** [comment]
@teleivo I made that variable final. I am hoping your feedback. Thanks

**Comment by dilekamadushan** [comment]
@teleivo Thanks For Merging the PR. Feel free to review on #2536 and #2541

**Comment by teleivo** [comment]
@dilekamadushan no need to reference your PRs when we merge one. We can see all PRs that are open and will go through them one by one. We are only a few maintainers. Please read the issues you take on carefully and work through them before taking on a new one. Thank you very much for your efforts!

**Comment by dilekamadushan** [comment]
@teleivo Thank you so much for your help. Sorry if you were offended

**Comment by teleivo** [comment]
don't worry I am not offended. I just want to make sure contributors are also mindful of other contributors and the time of maintainers. Often people take up a lot of issues, block them and don't finish them or take up to many and only work a little without finishing them. which is not helpful. that's all. thank you very much! and happy new year :)

**Comment by dilekamadushan** [comment]
@teleivo Wish you a very happy new year too!. I would try my best to finish off all the issues I claimed. And thank you very much for your support which is invaluable.

**Pull request merged**